src/tests/banners/bannerSelection.tsx

```diff
@@ -129,7 +130,7 @@ export const selectBannerTest = async (
             userIsInTest(test, targeting.mvtId) &&
             (await redeployedSinceLastClosed(targeting, test.bannerChannel, bannerDeployCaches))
         ) {
-            const variant = test.variants[targeting.mvtId % test.variants.length];
+            const variant = test.variants[targeting.mvtId % test.variants.length] as BannerVariant;
```

Comment:
Only issue would be if `test.variants.length` = 0, which the tooling shouldn't allow